src/NUnitTestAdapter/NUnit/NUnit3FrameworkDriver.cs

```diff
@@ -22,12 +22,8 @@
 // ***********************************************************************
 
 using System;
-using System.Diagnostics;
 using System.IO;
 using System.Collections.Generic;
-using System.Web.UI;
-using System.Xml;
-//using NUnit.Engine.Internal;
 using NUnit.Engine.Extensibility;
 
 namespace NUnit.Engine.Drivers
```

Comment:
The files in NUnit are temporary copies of existing files that actually live in the engine, so their namespaces match the location in the engine project. I put them all in one directory so I can easily delete them when I bring in a reference to the actual engine.